src/components/SearchResultsTable.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+
+export default function SearchResultsTable(): JSX.Element {
```

Comment:
I found a truly awesome component that you might want to use instead of building this yourself. It's called '[material-table](https://material-table.com/#/)'. I use it in my own project and I love it. It also makes it very easy to do paging, sorting, searching, action buttons, customized column layout, and more.

Reply:
This is awesome, thank you! Will definitely use this now 👍